### ISSUE TITLE
[Feature request] some improvements

### ISSUE THREAD

**Issue by BachiMjavanadze** [ManuelGil/vscode-angular-generator#5]
At the moment, to generate a component or something else, I need to call the context menu from a folder. Please add the ability to call the context menu from a file. In this case, the extension should intercept the path of the file's parent folder and generate the element (component, service, etc.) in it.

Also, at the moment, when I call the command to execute, I see an input field with the path, something like this:

`src/app/components/products`. 

Please create optional seetings:

` "angular.submenu.customCommands.showPath": true/false`

Of course the default value should be `true` but if I set to `false` then I should not see the path but under the hood path should be:

`src/app/components/products/` _ slash symbol at the end.

So I will be able to just write the identifier, without slash symbol.

**Comment by ManuelGil** [comment]
These updates are a bit out of the norm. I will be reviewing the possibility of implementing them, but I cannot assure their inclusion in a future version.

**Comment by ManuelGil** [comment]
Hi @BachiMjavanadze ,

I have made an update to integrate these improvements among other changes.

I look forward to your comments.

**Comment by BachiMjavanadze** [comment]
Hi @ManuelGil ,

after updating the extension I do not see ` Angular File Generator` menu item in the context menu. I've removed all my custom settings for the extension but result is the same. I'm on the latest stable `Windows 11` and and `VSCode`:

![image](https://github.com/user-attachments/assets/7d29603e-4403-4819-92e1-5e67dc087c06)

**Comment by ManuelGil** [comment]
The issue occurred because the activation event **/angular.json delayed the activation and the `angular.activateItem` configurations were not loaded in time to display the menu.

Please, try again.

**Comment by BachiMjavanadze** [comment]
@ManuelGil 
nothing changed for me, the same bug

**Comment by BachiMjavanadze** [comment]
in Version 2.9.3 problem has gone but can't reach designed result by settings:

` "angular.submenu.customCommands.showPath": true/false`.

Looks like the feature is not implemented.

**Comment by BachiMjavanadze** [comment]
no, problem is still here, even more: can't see drop down menu, just spinning on status bar:

![image](https://github.com/user-attachments/assets/14d86ef1-9d47-4e13-8ff4-f220641c3e2a)

**Comment by ManuelGil** [comment]
I am a bit worried because I will be traveling for two weeks and will not be available.

I have just uploaded an emergency update in the hope that it will completely fix the extension activation issue.

I hope this will be the final fix for this bug.

Happy New Year and happy holidays.

**Comment by BachiMjavanadze** [comment]
@ManuelGil
nothing changed for me, the same bug.
Happy New Year and happy holidays.
